### DISCUSSION TITLE
Window title/name

### DISCUSSION THREAD

**Discussion by drew-bombard** [Sequel-Ace/Sequel-Ace#1789] [Ideas]
Would LOVE a way to change the title of the window... instead of showing the DB type and details.

Example:
<img width="491" alt="Screenshot 2023-07-13 at 10 12 41 AM" src="https://github.com/Sequel-Ace/Sequel-Ace/assets/104865944/3fb4eff9-9c23-4482-9061-b68b73323a6b">

Can you tell from this screen grab which DB is my docker sandbox and which is Production?  Yeah, me either.  

It'd be great if the _NAME_ that I assign the DB profile appears first, and then I perhaps the DB type and details are smaller caption text underneath it.  Heck, even have badge or indicator that shows the same color you've applied to the connection favorite.  I know that the tabs contain the color strip at the bottom, but there is no indicator on non-tabbed windows.  Here's a quick mockup of what I'm thinking:

![mockup-production](https://github.com/Sequel-Ace/Sequel-Ace/assets/104865944/544419d8-89e2-435f-83fa-5a8a38b21989)
![mockup-staging](https://github.com/Sequel-Ace/Sequel-Ace/assets/104865944/efe4e568-71ae-4224-b697-6f5e6de66c3d)
![mockup-docker](https://github.com/Sequel-Ace/Sequel-Ace/assets/104865944/f840afa7-ce85-4d38-898d-1d429e8e18a4)

**Comment by samuelbradshaw** [comment]
I agree – I don't usually have a need to differentiate between production and test servers, but I would rather see my database name than the MySQL version. The MySQL version isn't usually what I'm looking for or care about – that could be hidden away in a side menu (or in a subtitle, as you mocked up – but I'm not sure if that's possible using standard OS components).

**Reply by samuelbradshaw** [reply]
Related: https://github.com/Sequel-Ace/Sequel-Ace/issues/1195, https://github.com/Sequel-Ace/Sequel-Ace/issues/1083
The workaround given is to use tabs, which show the database name and a color. But I usually prefer using separate windows so I can have multiple databases up side by side. Adding a tab bar takes up vertical space.

**Reply by drew-bombard** [reply]
Exactly @samuelbradshaw... totally agree.  The amount of times I need to know the DB I'm connected to, vs. the version of the DB I'm on is 1,000 to 1.

Heck, the color would be nice, but sure... not a _huge_ deal to continue using tabs.  If that's the case, it'd be nice to have a preference to **always** show the tab bar.  Anyway... the name is the biggest change that I think would benefit the most people.
